test/e2e/spinup_test.go

```diff
@@ -143,8 +142,7 @@ func receiver(i int, config string) cmdScheduleFunc {
 		return append(cmds, newCmdExec(exec.Command("thanos", "receive",
 			"--debug.name", fmt.Sprintf("remote-write-receive-%d", i),
 			"--grpc-address", remoteWriteReceiveGRPC(i),
-			"--http-address", remoteWriteReceiveMetricHTTP(i),
-			"--remote-write.address", remoteWriteReceiveHTTP(i),
```

Comment:
ditto

Reply:
not sure what do you mean here if this is ping for brancz or did I make something wrong here?

Reply:
this part lgtm